### PR TITLE
Port CLI docs

### PR DESCRIPTION
**Pull request by jerivas**
Visit https://deploy-preview-49--sass-site-oddbird.netlify.app/documentation/cli/

### CODE REVIEW

File: source/_includes/compatibility.liquid [hunk 1]
```diff
@@ -2,40 +2,40 @@
   <div class="compatibility">
     Compatibility{% if feature %} ({{ feature }}){% endif %}:
   </div>
-  {% if dart != null %}
+  {%- if dart != null -%}
```
Comment (jerivas) [inline source/_includes/compatibility.liquid]: This fixes whitespace being interepreted as an empty `<p>` and creating extra height in the compatibility block

File: source/helpers/engines.ts [hunk 2]
```diff
@@ -7,12 +7,25 @@ import path from 'path';
 
 import { renderPermalink } from './components/anchors';
 
+/**
+ * Identical to `markdown-it-anchor`'s default slugify function, but removes
+ * leading dashes to match the behavior of the old Ruby site.
```
Comment (jerivas) [inline source/helpers/engines.ts]: In the current site, a section with a heading like[ `<h3>--quiet</h3>`](https://sass-lang.com/documentation/cli/dart-sass#quiet) would produce an anchor link to `#quiet` (leading dashes omitted). In the new site, `markdown-it-anchor` was keeping the dashes so the anchors were being generated as `#--quiet`, which breaks old links. This change preserves the old behavior.

**Comment by jerivas** [comment]
@jgerigmeyer @sanajaved7 I created `.liquid` files for consistency with the other PRs, but I think we could use `.md` to avoid the `{% markdown %}` block altogether. What do you think?

**Comment by jerivas** [comment]
Hmm, looks like Netlify deploy previews are not enabled for branches not based off of `main`. For some reason `yarn lint` is also not running

**Comment by jerivas** [comment]
[port documentation](https://trello.com/c/aNIO6Sga/29-port-documentation)

**Comment by sanajaved7** [comment]
> Hmm, looks like Netlify deploy previews are not enabled for branches not based off of `main`. For some reason `yarn lint` is also not running

@jerivas @jgerigmeyer is that a setting we can adjust to enable the deploy preview for these branches? It would definitely make reviewing easier.

**Comment by jerivas** [comment]
@sanajaved7 just figured out how to do it in the Netlify settings. Previews should be working now

File: source/helpers/engines.ts (hunk 2 above)
Comment (jgerigmeyer) [reply source/helpers/engines.ts]: I resolved this similarly in #50 by using https://github.com/sindresorhus/slugify.

**Comment by jgerigmeyer** [comment]
> @jgerigmeyer @sanajaved7 I created `.liquid` files for consistency with the other PRs, but I think we could use `.md` to avoid the `{% markdown %}` block altogether. What do you think?

@jerivas I haven't reviewed this PR yet, but my general approach has been that it's okay to use `.md` if the entire file is actual markdown.

**Review by jgerigmeyer — APPROVED** [review]
> @jgerigmeyer @sanajaved7 I created `.liquid` files for consistency with the other PRs, but I think we could use `.md` to avoid the `{% markdown %}` block altogether. What do you think?

@jerivas I went back and forth on this... On one hand I worry about embedding the markup partials (`{% compatibility %}` and `{% headsUp %}`) inside markdown, but on the other hand it does simplify these files and we've already done the work of making them render fine within markdown... So I went with `.md` files for these, but open to feedback.

**Pull request merged**